public/manifest.json

```diff
@@ -32,7 +32,7 @@
     "notifications",
     "tabs"
   ],
-  "content_security_policy": "script-src 'self' 'wasm-eval'; object-src 'self'",
+  "content_security_policy": "default-src 'self'; frame-ancestors 'none';",
```

Comment:
This was extracted from https://github.com/oasisprotocol/oasis-wallet-ext/pull/75/commits/777b410dfa4ed2c50459ba9e418529c56a1e9c6b#diff-639916bc14c3f311c31f629a2ec116292a4b2f64d06b3607b9ddd2e495703895L35-R35

but the next commit loosened CSP again https://github.com/oasisprotocol/oasis-wallet-ext/pull/75/commits/63fe8298f7a908e3a15fb167507e5c12c81e9bbe#diff-639916bc14c3f311c31f629a2ec116292a4b2f64d06b3607b9ddd2e495703895L35-R35

because it uses iframes https://github.com/oasisprotocol/oasis-wallet-ext/issues/6#issuecomment-901559677

Reply:
Ok, I've removed `frame-ancestors 'none';`.

I would prefer to keep `default-src 'self';` and only override the directive that's needed for CSS to work as per https://github.com/oasisprotocol/oasis-wallet-ext/issues/6#issuecomment-901559677.